### PR TITLE
fix server.go:243:6: no new variables on left side of :=

### PR DESCRIPTION
**Pull request by zonghuili2019**
redefine tgr

### CODE REVIEW

**Comment by Cyberhan123** [comment]
yes, I got same

**Review by SagarNirmanlab — APPROVED** [review]

**Review by MilanSavaliya — APPROVED** [review]

File: server/server.go [hunk 1]
```diff
@@ -240,7 +240,7 @@ func (s *Server) GetAuthorizeToken(ctx context.Context, req *AuthorizeRequest) (
 		}
 	}
 
-	tgr := &oauth2.TokenGenerateRequest{
+	tgr = &oauth2.TokenGenerateRequest{
```
Comment (MilanSavaliya) [inline server/server.go:243]: perfect. :ok_hand:

**Review by MilanSavaliya — APPROVED** [review]

**Comment by MilanSavaliya** [comment]
I think it would be good if we can merge this as soon as possible.

**Comment by arran4** [comment]
Hey, I don't have write access to the repo sorry.

**Comment by MilanSavaliya** [comment]
@LyricTian is it possible to create new version tag and make it available ?

**Comment by Cyberhan123** [comment]
> @LyricTian is it possible to create new version tag and make it available ?

May not, It's an hot fix ,so version tag may  an bad idea. cc @LyricTian How do you see?

**Comment by novrian** [comment]
@Cyberhan123 v4 is broken right now.
If it's not possible to create a new tag then how can we use v4?

**Comment by Cyberhan123** [comment]
> @Cyberhan123 v4 is broken right now.
> If it's not possible to create a new tag then how can we use v4?

It just an cache, U can run 
`
go get -u -v github.com/go-oauth2/oauth2/v4/...
`
uninstall it and go get fetch it again

**Comment by novrian** [comment]
Screen Shot 2021-07-16 at 14 08 11

@Cyberhan123 no luck, still experience the same error

nevermind, maybe I will use v3 either. thanks for helping han

**Comment by MilanSavaliya** [comment]
@Cyberhan123 @LyricTian i think this will not work as releases are tide with particular commit hash as you can see in attached picture. You have to create newer tag release to offer this fix. 

image

**Comment by Cyberhan123** [comment]
> @Cyber​​han123 @LyricTian我认为这不会起作用，因为发布与特定的提交哈希相关联，如您在附图中所见。您必须创建更新的标签版本才能提供此修复程序。
> 
> 图像



> 屏幕截图 2021-07-16 at 14 08 11
> 
> @Cyber​​han123不走运，仍然遇到同样的错误
> 
> 没关系，也许我也会使用 v3。谢谢你帮助韩



> @Cyberhan123 @LyricTian i think this will not work as releases are tied with particular commit hash as you can see in attached picture. You have to create newer tag release to offer this fix.
> 
> image

oh, your right and I want check out why my packge are worked

**Pull request merged**